react/__tests__/ProductList.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { render, fireEvent } from '@vtex/test-tools/react'
+
+import ProductList from '../ProductList'
+
+describe('Product List', () => {
+  const mockItems: Item[] = [
```

Comment:
You should extract this code to a `__MOCKS__/mockItems`, so it's easier to read tests.